lib/plugins/inspec-init/lib/inspec-init/cli_profile.rb

```diff
@@ -24,14 +24,24 @@ def valid_profile_platforms
              desc: "Which platform to generate a profile for: choose from #{valid_profile_platforms.join(", ")}"
       option :overwrite, type: :boolean, default: false,
              desc: "Overwrites existing directory"
+      option :resource_dependency, default: nil, type: :string, aliases: [:r],
```

Comment:
Are we supporting passing multiple resource pack dependencies in this? If not then simply `aliases: :r` should be used here.